src/xtgeo/grid3d/grid_properties.py

```diff
@@ -119,23 +331,21 @@ def names(self):
             ...     print ('Property name is {}'.format(name))
             Property name is PERMX
 
-            >>> # when setting names, an empty string 'trick'
-            >>> # will make the namelist to be defaulted as
-            >>> # each individual property name
-
-            >>> props.names = ""
-
         """
         return self._names
 
     @names.setter
+    @deprecation.deprecated(
+        deprecated_in="2.16",
+        removed_in="4.0",
+        current_version=xtgeo.version,
+        details="The behavior of the setting names would before create an alias name "
+        "the behavior of which was not always consistent."
+        "This name aliasing is now going away. "
+        "In order to change the name of properties, "
+        "use\nfor p in gridprops:\n    p.name = newname",
+    )
```

Comment:
I believe this message should clearly state that this call has no effect / is ignored

Reply:
Done